src/symtable/SymbolTableScope.ts

```diff
@@ -141,9 +141,14 @@ export class SymbolTableScope {
     }
 
     _identsMatching(f: (flag: number) => boolean): string[] {
-        return Object.entries(this.symbols)
-            .filter(([_, v]) => f(v))
-            .map(([k, _]) => k);
+        const idents = [];
```

Comment:
it looked like this before I changed it to the `filter map` haha

Reply:
`Object.entries` is just quite slow unfortunately.

Reply:
And also javascript iterators I guess.